payloads/external/Makefile.inc

```diff
@@ -165,26 +165,43 @@ $(obj)/UEFIPAYLOAD.fd: $(DOTCONFIG) $(IPXE_EFI)
 		CONFIG_EDK2_REPOSITORY=$(CONFIG_EDK2_REPOSITORY) \
 		CONFIG_EDK2_TAG_OR_REV=$(CONFIG_EDK2_TAG_OR_REV) \
 		CONFIG_EDK2_UEFIPAYLOAD=$(CONFIG_EDK2_UEFIPAYLOAD) \
-		CONFIG_EDK2_REPO_OFFICIAL=$(CONFIG_EDK2_REPO_OFFICIAL) \
-		CONFIG_EDK2_REPO_MRCHROMEBOX=$(CONFIG_EDK2_REPO_MRCHROMEBOX) \
-		CONFIG_EDK2_REPO_CUSTOM=$(CONFIG_EDK2_REPO_CUSTOM) \
```

Comment:
Why is this removed?

Reply:
What happend is I've copied all these params from other protectli board since some were missing and went over the ones that were deleted by the "copy-paste". The thing is, these three options are in kconfig, but are not used in makefile, which is why I've pushed the change the way it is. For example same goes for our ehl vault branch.